ui/src/main/java/org/pentaho/di/ui/spoon/trans/TransGraph.java

```diff
@@ -4154,7 +4154,13 @@ public void run() {
   }
 
   public void stop() {
-    if ( ( running && !halting ) || safeStopping ) {
+    if ( safeStopping ) {
+      MessageBox messageBox = new MessageBox( shell, SWT.ICON_ERROR | SWT.OK );
```

Comment:
Should we use SimpleMessageDialog instead?

Reply:
Prolly

Reply:
The other message dialogs in TransGraph are all MessageBox, so I was going for consistency.  I'll send a screenshot to Stephanie to make sure it looks okay to her.

Reply:
ok.  :+1: